pkg/metrics/resource.go

```diff
@@ -67,6 +69,16 @@ var (
 		Buckets: prometheus.ExponentialBuckets(0.001, 2, 12),
 	}, []string{"result"})
 
+	recreateResourceWhenSyncWorkStatus = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: recreateResourceToCluster,
+		Help: "Number of recreating operation of the resource to a target member cluster. By the result, 'error' means a resource recreated failed. Otherwise 'success'. GVK means the group/kind/version of the resource. Cluster means the target member cluster.",
+	}, []string{"result", "gvk", "cluster"})
```

Comment:
the value of `gvk` looks like `apps/v1, Kind=Deployment`. such values ​​are too complex to be used for label filtering.
`apiVersion` and `kind` are recommended.

Reply:
thanks